src/components/DegreeChipStrip.css

```
@@ -1,97 +1,170 @@
+  font-weight: var(--font-weight-semibold);
+  color: var(--text-soft-white);
+  line-height: 1;
+  letter-spacing: 0.02em;
 }
 
-.degree-chip[data-is-tonic='true'] .degree-chip-note,
-.degree-chip[data-in-chord='true'] .degree-chip-note {
-  color: var(--chip-text-tonic);
+.app-container[data-layout-tier="mobile"] .degree-chip-strip {
+  --chip-size: 2.15rem;
+  width: 100%;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)